### DISCUSSION TITLE
Plugin Installation Error with version 1.1.23.1570

### DISCUSSION THREAD

**Discussion by 0x1f57** [community/community#17250] [Copilot Conversations] [answered]
![image](https://user-images.githubusercontent.com/102740967/170166620-84a883c2-bf0b-4319-a5f6-b05cabfeffd8.png)
In CLion 2022.1.1, This rough break occurs when I restart the IDE after updating the plugin to the latest version.

**Comment by Ratrme** [comment]
me too

**Comment by jansorg** [comment]
Thanks for reporting! 
Can you tell if the file shown in the screenshot does actually exist on your disk?
If it does, are you able to execute it?

**Comment by jansorg** [comment, marked as answer]
Earlier versions of the plugin did not properly terminate the Copilot helper process when the IDE was shutdown or restarted.
Now, when the IDE tried to delete the old plugin at startup and then to install the new update, the .exe file could not be deleted and the error popup was shown. 
The fix is to terminate all running `copilot-agent-win.exe` processes after terminating your IDE. The easiest way to do that is to restart Windows after shutting down the IDE with the old version of Copilot.
The latest version of Copilot has already fixed the bug and the helper process should terminate as expected.

**Reply by 0x1f57** [reply]
Thanks, the plugin is already working

**Reply by melontaro** [reply]
tks
